src/Mod/Draft/importSVG.py

```diff
@@ -1205,7 +1205,7 @@ def export(exportList,filename):
 
         svg_export_style = FreeCAD.ParamGet("User parameter:BaseApp/Preferences/Mod/Draft").GetInt("svg_export_style")
         if svg_export_style != 0 and svg_export_style != 1:
-            FreeCAD.Console.PrintMessage("unknown svg export style, switching to Translated\n")
+            FreeCAD.Console.PrintMessage(translate("Unknown SVG export style, switching to Translated")+"\n")
```

Comment:
Is this correct?

Reply:
The translate function is undefined. This will make the svg importer unusable... This should be added at the top:

```
if FreeCAD.GuiUp:
    from DraftTools import translate
else:
    def translate(ctxt,txt):
        return txt
```

Reply:
added